src/scss/components/_gallery-item.scss

```diff
@@ -0,0 +1,62 @@
+/**
+  * @parent gaiden_css.components
+  * @stylesheet gaiden_css.components.gallery-item.css Gallery Item
+  *
+  * ## Services
+  * @demo demo/gaiden-css/components/gallery-item/demo.html
+**/
+.gallery-item {
+  width: 100%;
+  position: relative;
+  overflow: hidden;
+  border-radius: $borderRadius;
+  cursor: pointer;
+  transition: transform .2s linear;
+  will-change: transform;
+
+  &:hover {
+    transition: transform .2s linear;
+    transform: translateY(-7px) scale(1.03);
+  }
+
+  &__title {
+    width: 100%;
+    position: absolute;
+    bottom: 0;
+    border-radius: 0 0 $borderRadius $borderRadius;
+    overflow: hidden;
+
+    &::before {
+      width: 100%;
```

Comment:
Is this blur effect really necessary? On the Sketch, the background dont have this opacity and blur.

The repaint of this effect, associated with the translate, is very expensive to the browser.